### DISCUSSION TITLE
same syntax?  n:class vs. n:attr

### DISCUSSION THREAD

**Discussion by qwerin** [nette/latte#302] [General]
Why syntax diferent?
n:class="$bool ? myClass ,.."
n:attr="attr: $bool, ..."

**Comment by dg** [comment]
It is like classic function arguments https://www.php.net/manual/en/functions.arguments.php vs named arguments https://www.php.net/manual/en/functions.arguments.php#functions.named-arguments

- `class($arg1, $arg2, $cond ? $arg3 : $arg4)`
- `attr(name1: $arg1, name2: $arg2)`
